src/metabase/driver/sql/query_processor.clj

```diff
@@ -757,7 +757,7 @@
 (defmethod ->honeysql [:sql :-]
   [driver [_ & [first-arg & other-args :as args]]]
   (cond (interval? first-arg)
-        (throw (ex-info (tru "Interval as first argrument to subtraction is not allowed.")
```

Comment:
@jeff-bruemmer do we by any chance refer to intervals as proper nouns in cases like this?

Reply:
It's a common noun.